src/mut/glial_cell.cpp

```diff
@@ -1,19 +1,350 @@
+#include <assert.h>
```

Comment:
For other reviewers: the factorisation of the mutable API using the TTRee concept will be done in another review.